ONSdigital/eq-survey-runner
# Read schema files as UTF-8

**What**
Read the schema file as UTF-8, needed for the £ sign that is now in it.

**How to test**
1. Start the server
2. Browse to http://localhost:5000/questionnaire/1
3. Check the survey is rendered correctly

**Who can review**
Anyone apart from @warrenbailey
